packages/node_modules/@webex/plugin-meetings/src/constants.ts

```diff
@@ -754,9 +754,6 @@ export const PEER_CONNECTION_STATE = {
   FAILED: 'failed'
 };
 
-export const RTC_CONFIGURATION_FIREFOX = {iceServers: [], bundlePolicy: 'max-compat'};
```

Comment:
was this not getting used !?

Reply:
it's moved into `MediaUtil.createPeerConnection()`